### ISSUE TITLE
Timeline events no longer received for MacOS

### ISSUE THREAD

**Issue by kenzieschmoll** [dart-lang/sdk#40496]
Breaking change is here: https://dart-review.googlesource.com/c/sdk/+/131360

DevTools relies on Timeline events for features like the Timeline page and Network Profiler. After this change, we no longer receive timeline events from MacOs devices.

See message in Observatory:
<img width="1180" alt="Screen Shot 2020-02-06 at 1 12 30 PM" src="https://user-images.githubusercontent.com/43759233/73981661-8e48c380-48e7-11ea-85be-179a56c4e4b5.png">

It looks like the events are being forwarded, but they should at least be sent to both streams. The default behavior of `flutter run` should enable our tools to work without having to pass additional flags as workarounds.

**Comment by kenzieschmoll** [comment]
After talking to @bkonyi we figured out the issue. In DevTools, we were adding the `--trace-systrace` flag, which before the CL landed to forward those events, did not affect us. Removing that flag fixes the issue and we receive timeline events as expected.

We should add a note to the message that says, "If you'd like to see timeline events in Observatory, please disable the `--trace-systrace` flag in your <Insert platform specific config file here (e.g. MainFlutterWindow.swift for MacOS)>.

Ben, is there any way we can modify this flag at runtime? It would be useful if we could do this from the DevTools Timeline when `--trace-systrace` is enabled.

**Comment by bkonyi** [comment]
No, it's not currently possible to swap out timeline recorders at runtime. This is something that's theoretically possible but probably wouldn't be considered high priority at this point with the NNBD work going on.
